uint16 payloadMask
st_SPAT[] spatMsg 
st_BSM[] bsmMsg
st_RSA[] rsaMsg
st_EVA[] evaMsg
